samples/99_zil_interpreter/app/main.rb

```diff
@@ -11,7 +11,7 @@ def tick(args)
   # - Render history
   # - Collect input
   # - Call send_input with input when pressing Enter
-  $gtk.request_quit unless $interpreter.alive?
+  $gtk.request_quit unless $interpreter.alive? || $debug_interpreter
```

Comment:
Instead of those changes to main.... maybe just add an generic exception catcher around the basic loop in `ZIL_BUILTINS[:REPL]` to catch all unexpected error messages ;) so that the program will really only quit when typing exit and exiting the REPL on the official way